src/plugins/intel_gpu/src/graph/fully_connected.cpp

```diff
@@ -191,29 +191,54 @@ kernel_impl_params fully_connected_inst::get_fake_aligned_params(kernel_impl_par
     if (orig_input_layout.format == format::bfyx && orig_output_layout.format == format::bfyx && can_apply_fake_alignment) {
         auto updated_param = orig_impl_param;
 
-        auto batch_size = std::accumulate(input_shape.begin(),
+        auto input_batch_size = std::accumulate(input_shape.begin(),
                                           input_shape.end() - 1,
                                           size_t{1},
                                           std::multiplies<size_t>());
+        auto output_batch_size = std::accumulate(output_shape.begin(),
```

Comment:
What is the case where the output_batch_size != input_batch_size ? Basically (M, K) * (K, N) => (M, N), so the batch sizes of input and output should be identical..